ruma-client tests fail with default features

**Describe the bug**

`cargo test` fails with default features.

**How to Reproduce**

On branch main, at commit c614ca13e33ece142f956cc3e78e6e9beff5dad0:

```
$ cargo test
...
test crates/ruma-client/src/lib.rs - (line 36) - compile ... FAILED
...
---- crates/ruma-client/src/lib.rs - (line 36) stdout ----
error[E0433]: failed to resolve: could not find `Client` in `ruma_client`
   --> crates/ruma-client/src/lib.rs:42:27
    |
9   | let client = ruma_client::Client::builder()
    |                           ^^^^^^ could not find `Client` in `ruma_client`
...
```

If I run `cargo test --features="client-api"` it compiles and the tests pass.

--- a/crates/ruma-client/src/lib.rs
+++ b/crates/ruma-client/src/lib.rs
@@ -34,9 +34,9 @@
 //! application service that does not need to log in, but uses the access_token directly:
 //!
 //! ```no_run
-//! # type HttpClient = ruma_client::http_client::Dummy;
-//! #
+//! # #[cfg(feature = "client-api")]
 //! # async {
+//! # type HttpClient = ruma_client::http_client::Dummy;
 //! let homeserver_url = "https://example.com".to_owned();
 //! let client = ruma_client::Client::builder()
 //!     .homeserver_url(homeserver_url)
@@ -59,8 +59,9 @@
 //! For example:
 //!
 //! ```no_run
-//! # let homeserver_url = "https://example.com".to_owned();
+//! # #[cfg(feature = "client-api")]
 //! # async {
+//! # let homeserver_url = "https://example.com".to_owned();
 //! # let client = ruma_client::Client::builder()
 //! #     .homeserver_url(homeserver_url)
 //! #     .build::<ruma_client::http_client::Dummy>()

